cfy_cluster_manager/main.py

```diff
@@ -60,19 +61,28 @@ def __init__(self,
         self.hostname = hostname
         self.cert_path = cert_path
         self.key_path = key_path
+        self.type = node_name.split('-')[0]
+        self.installed = False
+        self.config_path = join(
+            '/etc/cloudify', '{}_config.yaml'.format(node_name))
+        self.unit_name = SYSTEMD_RUN_UNIT_NAME.format(self.type)
 
 
 def _exception_handler(type_, value, traceback):
-    exception_traceback = ''.join(format_exception(type_, value, traceback))
-    logger.exception(exception_traceback)
+    error = type_.__name__
+    if str(value):
+        error = '{0}: {1}'.format(error, str(value))
+    logger.error(error)
+    debug_traceback = ''.join(format_exception(type_, value, traceback))
+    logger.debug(debug_traceback)
```

Comment:
I changed it back because the traceback could get really verbose, so I thought it might be better to keep it as it was.